cerbero/build/recipe.py

```diff
@@ -232,7 +233,8 @@ def gen_library_file(self, output_dir=None):
                 implib = genlib.create(
                     os.path.join(self.config.prefix, dllpath),
                     self.config.target_arch,
-                    os.path.join(self.config.prefix, 'lib'))
+                    os.path.join(self.config.prefix, 'lib'),
+                    self.library_name_override)
```

Comment:
This is not going to work, it's iterating over a list filenames, and library_name_override can provide a single override. It should be a boolean that uses the filename instaed of the file path, but since it hard to configure it for a single library it's probably better to just override the gen_library_file int the recipe that needs the special handling